packages/components/rollup.config.mjs

```diff
@@ -27,7 +27,7 @@ const plugins = [
   // "app" tree. Things in here should also be in publicEntrypoints above, but
   // not everything in publicEntrypoints necessarily needs to go here.
   addon.appReexports([
-    'components/**/*.js',
+    'components/**/!(*types).js',
```

Comment:
if we are following this pattern for type location going forward, should I also update the helper and modifiers pattern below?

Reply:
Currently the hds modifiers and helpers live under the same folder(s). which folder structure are you envisioning?

Types next to the files:
```
packages/components/src/helpers/hds-link-to-models.ts
packages/components/src/helpers/hds-link-to-models.types.ts
packages/components/src/modifiers/hds-clipboard.ts
packages/components/src/modifiers/hds-clipboard.types.ts
```

Creation of parent folders (is it possible? does it work in Ember?):
```
packages/components/src/helpers/hds-link-to-models/index.ts
packages/components/src/helpers/hds-link-to-models/types.ts
packages/components/src/modifiers/hds-clipboard/index.ts
packages/components/src/modifiers/hds-clipboard/types.ts
```

Reply:
That is a very good question. I'm not sure what the answer is atm, and so maybe it's best not to address this now.

I'll try out the above options just to see what it looks like/what works. It would be _nice_ if it could be consistent across all components/modifiers/helpers/etc.

Reply:
I'm not super familiar with app reexports - if the types are removed does this mean that a consumer in a v1 addon can't import things from type files (like enums etc), or is that still possible since they are included in the public entrypoints above?

Reply:
Ya I believe that is handled by the `publicEntrypoints`. CUT currently [re-exports the types ](https://github.com/hashicorp/consul-ui-toolkit/blob/main/toolkit/package.json#L111)(whoops!) but we also have an [index file that imports and exports all the types](https://github.com/hashicorp/consul-ui-toolkit/blob/main/toolkit/src/types/index.ts). That file is not part of the `appReexports` but it is [accessible from cloud-ui.](https://github.com/hashicorp/cloud-ui/blob/9285425b1ade4c27ab1a20fa5ee4e33addddf160/engines/monitoring/addon/components/page/platform-logs/resource/index.ts#L19)

Reply:
`appReexports` is a compatability layer tat only needed for v1 apps with hbs/js, with current setup it would still export types

Reply:
We currently don't have type files for helpers and modifiers, so I wouldn't worry about it just now.